src/ui/public/vis/editors/default/default.html

```diff
@@ -1,5 +1,15 @@
-<div class="collapsible-sidebar">
+<div class="collapsible-sidebar" ng-class="{
+    'collapsible-sidebar-small': vis.type.editorConfig.defaultSize === 'small',
```

Comment:
Since we are kind of using more [BEM-ish CSS](https://github.com/elastic/kibana/blob/master/style_guides/css_style_guide.md#concepts) could we rename those into: `collapsible-sidebar--small`, `collapsible-sidebar--medium`, etc.